api/v1beta1/activemqartemis_types.go

```diff
@@ -286,6 +286,14 @@ type AcceptorType struct {
 	SuppressInternalManagementObjects *bool `json:"suppressInternalManagementObjects,omitempty"`
 	// Whether to let the acceptor to bind to all interfaces (0.0.0.0)
 	BindToAllInterfaces *bool `json:"bindToAllInterfaces,omitempty"`
+	// Type of keystore being used; "JKS", "JCEKS", "PKCS12", etc. Default is "JKS"
+	KeyStoreType string `json:"keyStoreType,omitempty"`
```

Comment:
I think we can live wothout the type...
the jdk can now peek/introspect in a file to figure out the type it needs to read.
does it provide some other value?

Reply:
peeking now to see if i can find a reference to this claim of introspection

Reply:
I did a quick sanity check on artemis main and it seems jks is the default type if nothing is configured. The JDK doing the right thing even if the type is configured wrong is https://bugs.openjdk.java.net/browse/JDK-8062552
In short, I think we can safely omit the keyStoreType

Reply:
See below for commit addition with removal of keyStoreType.